aten/src/ATen/core/Dimname.h

```diff
@@ -37,7 +37,7 @@ struct CAFFE2_API Dimname {
 
 using DimnameList = c10::ArrayRef<Dimname>;
 
-static Symbol kWildcard = Symbol::dimname("*");
+extern Symbol kWildcard;
```

Comment:
`kWildcard` is only really used in Dimname.cpp so we can just delete it from the header